app/views/candidate_interface/apply/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :title, t('page_titles.application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <% if @course.present? %>
+        <span class="govuk-caption-xl"><%= @course.name %> (<%= @course_code %>)</span>
```

Comment:
Is there a way to specify in the `Course` model which attributes it has? I wonder what happens if Find changes their response, would this result in a `NoMethodError` in this view?

Reply:
Maybe - we don't on `manage-courses-frontend` but I will see if `json_api_client` allows us to.

Reply:
This works fine for all attributes apart from the `provider_code` one... so I've decided to only do it for `name`.